packages/smart/test/amm-factory-test.ts

```diff
@@ -125,8 +125,8 @@ describe("AMMFactory", () => {
       secondMarketFactory.address,
       marketId,
       _outcome,
-      _setsInForCollateral,
-      setsToBurn
+      _shareTokensIn.map((m) => BigNumber.from(m)),
+      BigNumber.from(0)
```

Comment:
I'm worried about this being zero. The test of the shareTokensIn calculation is how close it gets to minSetsOut. So the minSetsOut value needs to match what we want the UI to calculate when it figures out slippage tolerance.

Reply:
Good call.